### DISCUSSION TITLE
Fans ramp up to 100% when shutting down, restarting or if alienfx isn't minimized and running.

### DISCUSSION THREAD

**Discussion by sw0277** [T-Troll/alienfx-tools#323] [Q&A] [answered]
Is it normal for the fans to ramp up to 100% when shutting down or restarting? They also ramp up to 100% if I close alienfx and don't let it run in the background. 

Dell G5 SE 5505

**Comment by T-Troll** [comment, marked as answer]
This can happen, and what's why:
My app remember active power mode then start, and restore it before quit/sleep/hibernate. So if your system was in performance/G-Mode before app start - yes, it will be like this.

You can change system mode:
- Quit app
- Open Administrator CMD
- Run `alienfan-cli setgmode=0 setpower=....". I don't remember how many power modes G5SE has, so you can use 0 for manual (or 1,2,3 - check detection metrics into the second line).
- System will be set to selected mode as default (maybe until power off on some systems)

So next app start will get this mode as a system and will restore it.

**Reply by sw0277** [reply]
Perfect, now it makes sense. This is an awesome replacement for AWCC and is much better in my opinion. Thanks for all the work you do.

EDIT: I just upgraded to the latest version 8.2.6 and the "Restore power mode" setting that was just added works great. Thanks for the quick response and solution!!

**Reply by T-Troll** [reply]
Well... It's an issue, in fact, so latest version have an option now to not restore power mode at quit.
You are welcome.
